test/SlowTests/Voron/Issues/RavenDB-12268.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.IO;
+using FastTests.Voron;
+using Sparrow;
+using Voron;
+using Xunit;
+
+namespace SlowTests.Voron.Issues
+{
+    public class RavenDB_12268 : StorageTest
+    {
+        [Fact]
+        public void UsedScratchBuffersArentDeleted()
+        {
+            // prevent the work of the global flushing behavior
+            Env.Options.MaxNumberOfPagesInJournalBeforeFlush = long.MaxValue;
```

Comment:
You can just set flushing to false, which is much better.
You are also creating the `Env` and then setting the options, that isn't supported.
`override` the customize options method

Reply:
done